Sources/StreamChatUI/Appearance+Images.swift

```diff
@@ -215,6 +215,13 @@ public extension Appearance {
                 return nil
             }
         }()
+        public var groupChatAvatarPlaceholder: UIImage? {
+            if #available(iOS 13.0, *) {
+                return UIImage(systemName: "person.2.fill")?.withRenderingMode(.alwaysTemplate)
```

Comment:
@jitendra-simform  Please remove `withRenderingMode` if it's does not required.

Reply:
@parth-simformsolutions  Chat channel list avatar placeholder changes removed , will create separate branch for avatar placeholder changes